hi_IN.lproj/Panel.strings

```diff
@@ -0,0 +1,487 @@
+/*
+ Panel.strings
+ translations for preference panel and other dialogs
+ 
+ by plutonium.239.811@gmail.com 
+ */
+
+/* Preference Panel */
+"Preferences" = "पसंद";
+
+/* General Panel */
+"General" = "आम";
+"Window Style" = "खिड़की की शैली";
+"Classic" = "क्लासिक";
+"Seamless" = "निर्बाध";
+"Seamless window only support OS X later than 10.10." = "निर्बाध विंडो केवल 10.10 से बाद में OS एक्स का समर्थन करती है।";
+"Themes" = "विषयों";
+"Open Theme Folder" = "ओपन थीम फ़ोल्डर";
+"Please check `help`->`Quick Start` for short introduction. Visit http://theme.typora.io to get more themes or instructions." = "लघु परिचय के लिए कृपया `सहायता`->` त्वरित शुरुआत 'की जांच करें। अधिक थीम या निर्देश प्राप्त करने के लिए http://theme.typora.io पर जाएं।";
+"Word Count" = "शब्द गणना";
+"Always Show Word Count" = "हमेशा शब्द गिनती दिखाएं";
+"Font Size" = "फ़ॉन्ट आकार";
+"Auto ( Recommended )" = "ऑटो (अनुशंसित)";
+"Customized" = "स्वनिर्धारित";
+"If custom font size is used, the actual font size will depend on the base font size and css style." = "यदि कस्टम फ़ॉन्ट आकार का उपयोग किया जाता है, तो वास्तविक फ़ॉन्ट आकार बेस फ़ॉन्ट आकार और सीएसएस शैली पर निर्भर करेगा।";
+"Left Panel" = "बाएं पैनल";
+"Show Outline Panel by Default" = "डिफ़ॉल्ट रूप से आउटलाइन पैनल दिखाएं";
+"Collapsible Outline on Left Panel" = "बाएं पैनल पर संकुचित रूपरेखा";
+"Save without asking when\nswitch files on side panel" = "बिना पूछे बिना सहेजें जब साइड पैनल पर \nswitch फ़ाइलें";
+"Edited files will be saved without asking when switch files in left panel. \nWhen \"Ask to keep changes when closing documents\"बाएं पैनल में फ़ाइलों को स्विच करते समय संपादित फ़ाइलों को सहेजा जाएगा। \nWhen \" = "Edited files will be saved without asking when switch files in left panel. \nWhen \"Ask to keep changes when closing documents\" is disabled in System Preference, this is always enabled.";
```

Comment:
This line is also not correct translated

Reply:
Should be
```
"Edited files will be saved without asking when switch files in left panel. \nWhen \"Ask to keep changes when closing documents\" is disabled in System Preference, this is always enabled." = "बाएं पैनल में फ़ाइलों को स्विच करते समय संपादित फ़ाइलों को सहेजा जाएगा। \nजब सिस्टम वरीयता में \"Ask to keep changes when closing documents\" अक्षम होता है, तो यह हमेशा सक्षम होता है।"
```